prow/cluster/monitoring/mixins/prometheus/ghproxy_alerts.libsonnet

```diff
@@ -5,21 +5,29 @@
         name: 'ghproxy',
         rules: [
           {
-            alert: 'ghproxy-status-code-abnormal-%sXX' % code_prefix,
-            // excluding 404 because it does not indicate any error in the system
+            alert: 'ghproxy-specific-status-code-abnormal',
             expr: |||
-              sum(rate(github_request_duration_count{status=~"%s..", status!="404"}[5m])) / sum(rate(github_request_duration_count{status!="404"}[5m])) * 100 > 5
-            ||| % code_prefix,
+              sum(rate(github_request_duration_count{status=~"[45]..",status!="404",status!="410"}[5m])) by (status,path) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) by (path) * 100 > 10
+            |||,
             labels: {
               severity: 'slack',
             },
             annotations: {
-              message: 'ghproxy has {{ $value | humanize }}%% of status code %sXX in the last 5 minutes.' % code_prefix,
+              message: 'More than 10% of all requests for {{ $labels.path }} through the GitHub proxy are errorring with code {{ $labels.status }}.',
             },
-          }
-          for code_prefix in ['4', '5']
-        ] +
-        [
+          },
+          {
+            alert: 'ghproxy-global-status-code-abnormal',
+            expr: |||
+              sum(rate(github_request_duration_count{status=~"[45]..",status!="404",status!="410"}[5m])) by (status) / ignoring(status) group_left sum(rate(github_request_duration_count[5m])) * 100 > 3
```

Comment:
I'm not sure that we should be making the alert even more sensitive, especially without increasing the time window.

Reply:
I checked the metrics we have in history and this doesn't trip in a false positive, so why?